magda-web-client/src/Pages/HomePageComponents/StoryBox.js

```diff
@@ -25,9 +26,9 @@ class StoryBox extends Component {
     getClickableElement(el, url) {
         if (!url) return el;
         return (
-            <a target="_blank" href={url} className="story-title-link">
+            <Link target="_blank" to={url} className="story-title-link">
```

Comment:
This link is actually the link used for stories, like this one:
![image](https://user-images.githubusercontent.com/13863530/39414453-f149cf80-4c7a-11e8-9cda-0cf5dea89583.png)